src/transformers/modeling_utils.py

```diff
@@ -1913,7 +1913,19 @@ def float(self, *args):
             return super().float(*args)
 
     @classmethod
-    def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.PathLike]], *model_args, **kwargs):
+    def from_pretrained(
+        cls,
+        pretrained_model_name_or_path: Optional[Union[str, os.PathLike]],
+        *model_args,
+        config: Optional[Union[PretrainedConfig, str, os.PathLike]] = None,
+        cache_dir: Optional[Union[str, os.PathLike]] = None,
+        force_download: bool = False,
+        resume_download: bool = False,
+        local_files_only: bool = False,
+        use_auth_token: Optional[Union[str, bool]] = None,
```

Comment:
This one should probably be renamed as `token` and only take a `str` (we will keep `use_auth_token` for backward comp of course, but since we expose things) that would be more consistent with `hf_hub` right @Wauplin ?

Reply:
Yes agree. In `huggingface_hub` the terminology is now `token` everywhere no matter if it's a string or a bool.